src/components/InvoicesList/InvoicesList.js

```diff
@@ -0,0 +1,113 @@
+        setStateInvoices({ loading: false, items: [] });
+      } else {
+        setStateInvoices({
+          loading: false,
+          items: invoicesResponse.value.items,
+          totalItems: invoicesResponse.value.totalItems,
+        });
+      }
+    };
+
+    fetchData();
+  }, [dopplerBillingApiClient]);
+
+  return (
+    <>
+      <Helmet>
+        <title>{'Invoices'}</title>
+        <meta name="invoices" />
+      </Helmet>
+      <HeaderSection>
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <h2>
+            <FormattedMessage id="invoices_list.title" />
+          </h2>
+          <p>
+            <FormattedMessage id="invoices_list.sub_title" />
+          </p>
+        </div>
+      </HeaderSection>
+      <section className="dp-container">
+        {/* <div> */}
+        <div className="dp-table-responsive">
+          {stateInvoices.loading ? (
+            <Loading page />
+          ) : (
+            <table className="dp-c-table">
+              <thead>
+                <tr>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.account_id_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.product_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.date_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.currency_column" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="invoices_list.amount_column" />
+                  </th>
+                  <th scope="col"></th>
+                </tr>
+              </thead>
+              <tbody>
+                {stateInvoices.items.length ? (
+                  <>
+                    {stateInvoices.items.map((invoice, index) => {
+                      return (
+                        <tr key={index}>
+                          <td>{invoice.accountId}</td>
+                          <td>{invoice.product}</td>
+                          <td>{invoice.date}</td>
+                          <td>{invoice.currency}</td>
+                          <td>{invoice.amount}</td>
+                          <td>
+                            <FormattedMessage id="invoices_list.download_msg" />
+                          </td>
+                        </tr>
+                      );
+                    })}
+                  </>
+                ) : (
+                  <tr>
+                    <td>
+                      <span className="bounceIn">
+                        <FormattedMessage id="invoices_list.no_data_msg" />
+                      </span>
+                    </td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
```

Comment:
What are these empty elements?

Reply:
Oh, it is a kind of patch because you do not have more columns. Maybe we do not need the table when there are not elements. @GusBaamonde how are we dealing with it in other cases?

Reply:
We already reviewed the case with Juan, he will apply a simple solution. The empty table is shown saying that "there is no data available" since the design does not contemplate this case. It can be reviewed with design later.